.github/workflows/codeql.yaml

```diff
@@ -27,18 +28,18 @@ jobs:
           build-mode: autobuild
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v5
+      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8  # v5.0.0
       with:
         persist-credentials: false
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v4
+      uses: github/codeql-action/init@0499de31b99561a6d14a36a5f662c2a54f91beee  # v4.31.2
```

Comment:
About [yamllint issue](https://github.com/yaml/go-yaml/actions/runs/18986464360/job/54231143737#step:3:1) reported by the CI 

You can read this

> Using a yaml directive to disable next line should as be considered as it's an easy move, that imply nothing else on the yamllint configuration.
>
> `# yamllint disable-line rule:line-length` looks like it does, so. Then no changes need to be made to `.yamllint`.
_Originally posted by @esacteksab in https://github.com/yaml/go-yaml/issues/118#issuecomment-3288653091_

Reply:
yeah, I read that

Reply:
changed to per-line comment